guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,6 +12,13 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you run {Project} in a disconnected environment, the following additional prerequisites must be met:
+  - You must obtain and deploy Leapp metadata manually. For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+  - You require access to {RHEL} and {Project} packages.
+* Follow the steps for obtaining ISO files for {RHEL} 8 and {Project} in xref:upgrading_a_disconnected_satellite[].
+* Configure the `--enablerepo <id>` file specific with each repository.
+* When running `leapp preupgrade` and `leapp upgrade`, you must pass `--enablerepo <id>` for each manually defined repository from the previous step.
+For more information on customizing the Leapp upgrade for your environment, see https://access.redhat.com/articles/4977891[Customizing your {RHEL} in-place upgrade].
```

Comment:
```suggestion
* If you run {Project} in a disconnected environment, the following additional prerequisites must be met:
  - You must obtain and deploy Leapp metadata manually. For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
  - You require access to {RHEL} and {Project} packages.
  - Follow the steps for obtaining ISO files for {RHEL} 8 and {Project} in xref:upgrading_a_disconnected_satellite[].
* For more information on customizing the Leapp upgrade for your environment, see https://access.redhat.com/articles/4977891[Customizing your {RHEL} in-place upgrade].
```

I dropped the `--enablerepo` lines, as that is not a *pre*requisite, and is described in detail below.

Also made "Fore more info" an own point, as otherewise it rendered weirdly.